tests/integration/otel_thrift_tests.py

```diff
@@ -220,12 +220,14 @@ def example(self, context):
             thrift_client_span.get_span_context().span_id,
         )
 
-        self.assertEqual(thrift_client_span.name, "example_service/example")
+        self.assertEqual(
+            thrift_client_span.name, "tests.integration.test_thrift.TestService/example"
```

Comment:
So this contains the full python package name. To get something close to a "thrift" package, you'll need to only keep the last part of the package. 

The assertion should be against `test_thrift.TestService`.

Reply:
Sounds good, is it always only a single depth package? I should be able to just split on `.` and take the last element.